generators/csharp/sdk/src/endpoint/snippets/SnippetsJsonGenerator.ts

```diff
@@ -0,0 +1,92 @@
+import { FernGeneratorExec } from "@fern-fern/generator-exec-sdk";
+import { Endpoint } from "@fern-fern/generator-exec-sdk/api";
+import { ExampleEndpointCall, HttpEndpoint } from "@fern-fern/ir-sdk/api";
+import { SdkGeneratorContext } from "../../SdkGeneratorContext";
+import { GrpcEndpointGenerator } from "../grpc/GrpcEndpointGenerator";
+import { HttpEndpointGenerator } from "../http/HttpEndpointGenerator";
+import urlJoin from "url-join";
+import { RootClientGenerator } from "../../root-client/RootClientGenerator";
+
+export interface EndpointSnippets {
+    autogenerated: SingleEndpointSnippet[];
+    userSpecified: SingleEndpointSnippet[];
+}
+
+export interface SingleEndpointSnippet {
+    imports: string | undefined;
+    endpointCall: string;
+}
+
+export class SnippetsJsonGenerator {
```

Comment:
nit: The file it produces is called `snippet.json` so we can update this to match.
```suggestion
export class SnippetJsonGenerator {
```